pkg/systemd/overview-cards/shutdownStatus.jsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React, { useState, useEffect } from 'react';
+import { Button, Flex, FlexItem } from '@patternfly/react-core';
+import { PowerOffIcon, RedoIcon } from "@patternfly/react-icons";
+
+import * as timeformat from "timeformat";
+
+import cockpit from "cockpit";
+
+const _ = cockpit.gettext;
+
+const getScheduledShutdown = (setShutdownTime, setShutdownType) => {
+    const client = cockpit.dbus("org.freedesktop.login1");
+    return client.call("/org/freedesktop/login1", "org.freedesktop.DBus.Properties", "Get",
+                       ["org.freedesktop.login1.Manager", "ScheduledShutdown"], { type: "ss" })
+            .then(([result]) => {
+                if (result) {
```

Comment:
Would this ever be false if the call succeeds? It's a struct, after all. If there's any doubt, let's please log it in `else`.

Reply:
I'm not sure what our api dictates here? So I guess the if is useless and can be removed.

```
The returned value is a Promise that will complete successfully when the method returns, or fail if an error occurs.
```

Reply:
Exactly, I don't see how the response in a successful call would ever not be a valid struct (of course its *members* could be null, but the structure itself must be present.)